api/USERS_GUIDE.md

```diff
@@ -146,6 +146,17 @@ curl -i -X GET "https://www.pgdp.org/api/v1/projects?genre[]=Science&genre[]=Spo
     -H "X-API-KEY: $API_KEY"
 ```
 
+To limit which project fields are included, use the `field` parameter. Request
+multiple fields by appending `[]` to the parameter name and passing multiple
+of them.
```

Comment:
I don't think that's necessary as the prior examples demonstrate asking for the full project. I included it in the YAML since there aren't examples there to demonstrate with.